### DISCUSSION TITLE
issue with the drop-down window

### DISCUSSION THREAD

**Discussion by xfactor7788** [UMEP-dev/UMEP#345] [Q&A] [answered]
Hi, 
Although UMEP has been installed successfully as a plugin in QGIS 3.16 in my PC, an issue happened when I tried to generate a DSM with the "DSM generator" in Pre-Processor. when I click the bar to input a DEM, a drop-down window is expected to pop up, but it does not. Actually, the same issue exists in other toolboxes where the drop-down window appears.
Anyone else has any idea about this issue? How can I fix this? 
Many thanks!

**Comment by biglimp** [comment, marked as answer]
What format are your DEM in? It has to be a raster-based format recognisable by QGIS. Me myself prefer geotiff.

**Reply by xfactor7788** [reply]
Thanks, it's a tif format and I now realize that I made a mistake dealing with the dataset and resulting in the above-mentioned issue, which turns out there is nothing to do with the plugin.
